src/dos/program_more_output.cpp

```diff
@@ -275,15 +275,9 @@ MoreOutputBase::UserDecision MoreOutputBase::DisplaySingleStream()
 			}
 		}
 		if (!is_state_ansi && (current_column == previous_column) &&
-#if (CHAR_MIN == 0) // 'char' is unsigned
-		    (code >= ' ') &&
-#else // 'char' is signed
-		    (code >= ' ' || code < 0) &&
-#endif
-		    (code != code_del)) {
-			// Alphanumeric character outside of ANSI sequence
-			// always changes the current column - if not, the
-			// output must have been redirected
+		    is_extended_printable_ascii(code)) {
+			// Non-printable characters always change the current
```

Comment:
_Non-printable characters always change_ - opposite, printable characters. And _outside of ANSI sequence_ part is important.

Reply:
Thanks; will do.

Reply:
Let's original comment as-is.